### PR TITLE
API Renaming login to verify

### PR DESCRIPTION
**Pull request by ScopeyNZ**
Verify is a more appropriate name becuase verifying MFA might not necessarily be used for login.

### CODE REVIEW

File: client/src/boot/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ window.document.addEventListener('DOMContentLoaded', () => {
 
   const element = window.document.getElementById('mfa-app');
   const schemaURL = element.dataset.schemaurl;
-  ReactDOM.render(<MultiFactorApp schemaURL={schemaURL} />, element);
+  ReactDOM.render(<Login schemaURL={schemaURL} />, element);
```
Comment (robbieaverill) [inline client/src/boot/index.js]: Not sure I agree with this change. I'd expect a user to be able to go to the MFA endpoint at any point on the frontend, if they wanted to modify their MFA settings. It also doesn't look related to renaming the Login component

Comment (ScopeyNZ) [reply client/src/boot/index.js]: Ok I can revert this part. Right now `MultiFactorApp` is incredibly non-descriptive. I think a name like `Login` (given it's in an MFA namespace) is more descriptive as this handles the whole registration/verification flow presented while logging in.

**Comment by ScopeyNZ** [comment]
I'm aware of the failing test - was a little to eager to submit the PR.

File: client/src/boot/index.js (hunk 1 above)
Comment (robbieaverill) [reply client/src/boot/index.js]: I see, you've renamed that component as well. Ok I guess

File: client/src/containers/Login.js [hunk 2]
```diff
@@ -82,43 +90,34 @@ class MultiFactorApp extends Component {
...
+    const { schema, verificationCompleted } = this.state;
 
-    if (!schema || loginCompleted || !schema.registeredMethods.length) {
+    if (!schema || verificationCompleted || !schema.registeredMethods.length) {
       return null;
     }
 
     return (
-      <Login {...schema} onCompleteLogin={this.handleCompleteLogin} />
+      <Verify {...schema} onCompleteVerification={this.handleCompleteVerify} />
```
Comment (robbieaverill) [inline client/src/containers/Login.js]: Register has a store provided, does Verify need one too?

Comment (ScopeyNZ) [reply client/src/containers/Login.js]: No (not yet). Only `Register` controls its flow with Redux right now.

File: src/Model/RegisteredMethod.php [hunk 3]
```diff
@@ -47,11 +47,11 @@ public function getMethod()
     }
 
     /**
-     * @return LoginHandlerInterface
+     * @return VerifyHandlerInterface
      */
-    public function getLoginHandler()
+    public function getVerifyHandler()
```
Comment (robbieaverill) [inline src/Model/RegisteredMethod.php]: Side note: we should add a return type here
